src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,30 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Link, useLocation, useResolvedPath } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
```

Comment:
It is recommended to use an interface instead of a type for defining Props, as interfaces can provide better autocompletion and error checking in editors.